### PR TITLE
Added Brazilian Portuguese translation.

### PR DESCRIPTION
**Pull request by NotGigo**

### CODE REVIEW

**Comment by vecchiotom** [comment]
thanks for specifying the kind of portuguese :) most people don't even know brazilian portuguese and portuguese are different lol, heck most people still think brazilians speak spanish :^)

**Comment by blattersturm** [comment]
Hi - please update the PR using the new languages.ts file!

File: ext/cfx-ui/src/app/app.module.ts [hunk 1]
```diff
@@ -49,7 +49,8 @@ const l10nConfig: L10nConfig = {
 	locale: {
 		languages: [
 			{ code: 'en', dir: 'ltr' },
-			{ code: 'fr', dir: 'ltr' }
+			{ code: 'fr', dir: 'ltr' },
+			{ code: 'br', dir: 'ltr' }
```
Comment (Syntasu) [inline ext/cfx-ui/src/app/app.module.ts]: This states the ISO code for this is BR, meanwhile in the ISO-639 definitions this is known as the Brenton language. I think (not 100% sure) that [pt](https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?iso_639_1=pt) is more fitting.

File: ext/cfx-ui/src/assets/languages/locale-br.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    "#BottomNav_Home": "Início",
+    "#BottomNav_Servers": "Servidores",
+    "#BottomNav_Settings": "Configurações",
+    "#ServerList_HideFull": "Esconder Servidores Cheios",
+    "#ServerList_HideEmpty": "Esconder Servidores Vazios",
+    "#ServerList_MaxPing": "Latência Máxima",
+    "#Servers_Connecting": "Conectando",
+    "#Servers_ConnectingTo": "Conectando-se a {{ serverName }}...",
+    "#Servers_ConnectFailed": "Falha na conexão",
+    "#Servers_Error": "Ошибка (Erro)",
```
Comment (goustkor) [inline ext/cfx-ui/src/assets/languages/locale-br.json]: Translate "Ошибка" to "Erro."

**Comment by vecchiotom** [comment]
@Syntasu The problem is pt is not the same as Brazilian Portuguese, so if
someone wanted to add normal Portuguese, what would they use? It's
basically like English (UK) and English (US)

**Comment by summeryukata** [comment]
```pt is a language code for Portuguese, defined by ISO standards (see ISO 639-1 and ISO 3166-1 alpha-2). There is no ISO code for spoken or written Brazilian Portuguese.
[...]
pt-BR is a language code for the Brazilian Portuguese, defined by Internet standards (see IETF language tag).```

as seen on [wikipedia](https://en.wikipedia.org/wiki/Brazilian_Portuguese)

**Comment by NotGigo** [comment]
PR updated. Hopefully I didn't mess anything up.

**Pull request merged**